lib/SIL/SILVerifier.cpp

```diff
@@ -3546,15 +3546,25 @@ void SILModule::verify() const {
     g.verify();
   }
 
-  // Check all vtables.
+  // Check all vtables and vtable cache.
   llvm::DenseSet<ClassDecl*> vtableClasses;
+  size_t EntriesSZ = 0;
```

Comment:
No: the assert is outside of the loop, you need to sum-up all the entries from all the vtables *then* compare the overall sum to the cache size.